changelogs/fragments/7882-add-redfish-get-service-identification.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_command - Add redfish_info command to get service identification (https://github.com/ansible-collections/community.general/issues/7882)
```

The module this affects is `redfish_info`, not `redfish_command`, right?
```suggestion
  - redfish_info - add command ``GetServiceIdentification`` to get service identification (https://github.com/ansible-collections/community.general/issues/7882).
```